src/WarcraftLegacies.Source/UnitTypeTraits/SpellConduction/SpellConductionBuff.cs

```diff
@@ -61,4 +72,7 @@ private void DamageCaster(unit damager, float eventDamage) =>
       damagetype.Universal, @event.WeaponType);
```

Comment:
One of the things we discussed wanting to do was making Spell Conduction correctly take into account the resistances of the Eradicator. The reason it doesn't right now is this `damagetype.Universal`. If we change that to the damage type of the original damage, it will take the Eradicator's resistances into account.